code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -121,6 +121,8 @@
 
 	occupant_vis = new(null, src)
 	vis_contents += occupant_vis
+	if(airs[1])
+		airs[1].volume = CELL_VOLUME * 0.5
```

Comment:
why have this if check if it'll work?

Reply:
because tests are a pain 
![image](https://user-images.githubusercontent.com/42839747/158704724-c85314a5-9629-4741-ac88-5ba2fa31cbc5.png)
the check fails because `airs` is not initialized in the test so it comes out as null